keycloak.d.ts

```diff
@@ -413,7 +413,7 @@ declare namespace KeycloakConnect {
 
     loginUrl(uuid: string, redirectUrl: string): string
 
-    logoutUrl(redirectUrl: string): string
+    logoutUrl(redirectUrl: string, idTokenHint?: string): string
```

Comment:
If the `idTokenHint` is required as mentioned [here](https://github.com/keycloak/keycloak-nodejs-connect/pull/359/files#r837832033) then it should be required at all times. So I assume my previous comment will have to be reverted,
```suggestion
    logoutUrl(redirectUrl: string, idTokenHint: string): string
```

Reply:
Replied in the point below

Reply:
From my point of view, this is the only open question in this PR whether to change this signature or not. It affects only applications, which directly use `logourUrl` method.

- If we keep ?, it means that developers does not need to update their applications in case they directly use `logoutUrl` method. In this case, users will see "Logout confirmation" screen and will need to confirm logout. So it is change in the behaviour for the end users
- If we don't keep ? and make idTokenHint documented as mandatory parameter, it means that developers will need to update their applications in the case they directly use `logoutUrl` method (which I don't know how much of them is really using as described in the other comment...)